Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/checkins/qrcode/TraceLocation.kt

```diff
@@ -21,12 +23,45 @@ data class TraceLocation(
     val defaultCheckInLengthInMinutes: Int?,
     val cryptographicSeed: ByteString,
     val cnPublicKey: String,
-    val version: Int = TRACE_LOCATION_VERSION,
+    val version: Int = VERSION,
 ) : Parcelable {
 
+    /**
+     * Return a url for [TraceLocation] to be used as an input for [QrCodeGenerator]
+     * URL format https://e.coronawarn.app?v=1#QR_CODE_PAYLOAD_BASE64URL
+     */
+    @IgnoredOnParcel
+    val locationUrl: String by lazy {
+        val payloadBytes = qrCodePayload().toByteArray()
+        val base64Url = BaseEncoding.base64Url().omitPadding().encode(payloadBytes)
+        AUTHORITY.plus(base64Url)
+    }
+
+    /**
+     *  Returns a byte sequence that serves as an identifier for the trace location.
+     *  The ID is the byte representation of SHA-256 hash.
+     */
+    @IgnoredOnParcel
+    val locationId: ByteString by lazy {
+        val cwaDomain = CWA_GUID.toByteArray()
```

Comment:
Rename `cwaDomain` => `cwaGuid`?

Reply:
cwaDomain is the result of CWA_GUID.toByteArray() Please check specs as it says so: https://github.com/corona-warn-app/cwa-app-tech-spec/blob/6e45edc4df5a8ced091a179bcd40da9bcc162c85/docs/spec/event-registration-client.md#calculate-tracelocation-id

Reply:
okay :)